frontend-react/src/pages/details/Details.tsx

```diff
@@ -1,68 +1,30 @@
-import { NetworkErrorBoundary } from "rest-hooks";
-import { Suspense } from "react";
+import { useParams } from "react-router-dom";
 
 import HipaaNotice from "../../components/HipaaNotice";
-import Spinner from "../../components/Spinner";
-import { ErrorPage } from "../error/ErrorPage";
 import { useReportsDetail } from "../../hooks/network/History/DeliveryHooks";
 import { MemberType } from "../../hooks/UseOktaMemberships";
 import { AuthElement } from "../../components/AuthElement";
+import { withCatchAndSuspense } from "../../components/RSErrorBoundary";
 
 import Summary from "./Summary";
 import ReportDetails from "./ReportDetails";
 import FacilitiesTable from "./FacilitiesTable";
 
-/** Converts URL queries to a map-like object
- * @remarks Maybe this should make a real Map, not a map-like object? */
-function useQuery(): { readonly [key: string]: string } {
-    const query = window.location.search.slice(1);
-    const queryMap = {};
-    Object.assign(
-        queryMap,
-        ...query
-            .split(",")
-            .map((s) => s.split("="))
-            .map(([k, v]) => ({ [k]: v }))
-    );
-    return queryMap;
-}
-
-/** @todo Refactor as part of {@link https://github.com/CDCgov/prime-reportstream/issues/4790 #4790} */
 const DetailsContent = () => {
-    const queryMap = useQuery();
-    const reportId = queryMap?.["reportId"] || "";
-    const { data: report, loading } = useReportsDetail(reportId);
-    if (loading) return <Spinner size="fullpage" />;
+    const { reportId } = useParams();
+    const { reportDetail } = useReportsDetail(reportId!!);
 
     return (
         <>
-            <Summary report={report} />
-            <ReportDetails report={report} />
-            <NetworkErrorBoundary
-                fallbackComponent={() => <ErrorPage type="message" />}
-            >
-                <Suspense fallback={<Spinner />}>
-                    <FacilitiesTable reportId={reportId} />
-                </Suspense>
-            </NetworkErrorBoundary>
+            <Summary report={reportDetail} />
+            <ReportDetails report={reportDetail} />
```

Comment:
Follow-on work is going to restructure how we pass and call for data in here.

1. Report ID comes through params for the details page.
2. Pass Report ID to Summary (no call inside), ReportDetails (Details call inside), and FacilitiesTable (Facilities call inside) since all three _**ONLY**_ rely on the ID for display/network purposes.

Compared to now where we fetch the details, pass that JSON block into Summary where it isn't needed.